### DISCUSSION TITLE
Annotation problem

### DISCUSSION THREAD

**Discussion by Trainner111** [cafferychen777/ggpicrust2#83] [Q&A] [answered]
![image](https://github.com/cafferychen777/ggpicrust2/assets/138107559/f71c6370-486b-4983-b9f4-bc2117ea5de4)
after do pathway_annotion , there is no pathway description and all values are NA

**Comment by cafferychen777** [comment, marked as answer]
Dear User,

Regarding your issue with the `ggpicrust2` package where the pathway annotation results in missing descriptions and all values being NA, it's important to consider the network accessibility in your location, especially if you are in China. The functionality of the package may be affected by the Great Firewall, which can restrict access to certain online resources necessary for the package to operate correctly.

To resolve this issue, ensure that your network connection can bypass these restrictions. You might need to use VPN services or other methods to access the necessary online resources. Once you have a stable connection that circumvents the firewall, try running the pathway annotation function again.

If you continue to experience issues even after ensuring network accessibility, please provide more details about the error or the steps leading up to it, so we can assist you further.

Best regards,
Chen YANG
ggpicrust2 Package Development Team
